spec/jobs/rapid_ready_for_decision/disability_compensation_job_spec.rb

```diff
@@ -174,10 +194,23 @@
       end
 
       it 'raises an ArgumentError' do
-        Sidekiq::Testing.inline! do
-          expect do
+        VCR.use_cassette('evss/claims/claims_without_open_compensation_claims') do
+          Sidekiq::Testing.inline! do
+            expect do
+              subject.perform_async(submission.id)
+            end.to raise_error(ArgumentError, 'no ICN passed in for LH API request.')
+          end
+        end
+      end
+    end
+
+    context 'when there are pending claims that cause EP 400 errors' do
```

Comment:
I believe you mean:
```suggestion
    context 'when there are pending claims, which cause EP 400 errors' do
```
(since this test doesn't actually check for EP 400 errors; it checks for pending claims).